spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user do
+    
+  end
+end
```

Comment:
Surely a `User` needs more than that, in order to persist? This factory is incomplete.